examples/action/variables.tf

```diff
@@ -48,38 +29,45 @@ variable "resource_group_name" {
   type        = string
 }
 
-variable "provision_namespace" {
-  description = "Provision Namespace"
+##################################################
+# Action Variables
+##################################################
+variable "is_new_action" {
+  description = "Provision Action"
   type        = bool
-  default     = false
+  default     = true
 }
 
-variable "create_package" {
-  description = "Option whether to create a new package"
-  type        = bool
+variable "action_name" {
+  description = "Name of action."
+  type        = string
 }
 
-variable "package_name" {
-  description = "Package Name"
-  type        = string
+variable "action_limits" {
+  description = "Action runtime limits"
+  type        = list(any)
   default     = null
 }
 
-variable "package_publish" {
-  description = "Package visibilty."
+variable "action_exec" {
```

Comment:
expand the contents of action_exec like it has image, init, code, codepath etc

Reply:
Expanded action_exec